cmd/download.go

```diff
@@ -23,14 +27,18 @@ func addDownload(cmd *cobra.Command) {
 }
 
 func runFetch(cmd *cobra.Command, args []string) error {
+	// Retrieve flags
 	lockFile, err := cmd.Flags().GetString("lock-file")
 	if err != nil {
 		return err
 	}
+
+	// Open lock file as a local file, not a URL
 	lock, err := internal.NewLock(lockFile, false)
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to open lock file '%s': %w", lockFile, err)
```

Comment:
Is the error message that comes back from `internal.NetLock()` not sufficient?  If not, should it be addressed there, not here?